Require permission for the competitor to test the timer at the start of a BLD attempt. (#717)
thewca/wca-regulations: wca-guidelines.md

- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
- B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.

> - B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must ask (and receive an acknowledgement from) the judge before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without asking the judge: disqualification of the attempt (DNF).

Author: Per a suggestion by @Claster via Slack:

```suggestion
- B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).
```
